monetio/sat/_modis_l2_mm.py

```diff
@@ -80,8 +90,13 @@ def read_mfdataset(fnames, variable_dict, debug=False):
     if debug:
         logging_level = logging.DEBUG
         logging.basicConfig(stream=sys.stdout, level=logging_level)
+    else:
+        logging_level = logging.INFO
 
-    files = sorted(glob(fnames))
+    if isinstance(fnames, list):
+        files = fnames
+    else:
+        files = sorted(glob(fnames))
```

Comment:
bit more general this way
```suggestion
    if isinstance(fnames, str):
        files = sorted(glob(fnames))
    else:
        files = fnames
```